systest/backup/nfs-backup/docker-compose.yml

```diff
@@ -0,0 +1,344 @@
+      - 5080
+      - 6080
+    labels:
+      cluster: test
+      service: zero
+    volumes:
+
+      - type: bind
+        source: $GOPATH/bin
+        target: /gobin
+        read_only: true
+    command: /gobin/dgraph  ${COVERAGE_OUTPUT} zero --raft "idx=3;" --my=zero6_restore_clust_ha:5080 --replicas=3 --logtostderr -v=2 --bindall  --peer=zero4_restore_clust_ha:5080
+
+#non HA cluster backup cluster
+  zero7_backup_clust_non_ha:
+    image: dgraph/dgraph:local
+    working_dir: /data/zero7
+    links:
+      - "nfs:nfs"
+    labels:
+      cluster: test
+    ports:
+    - "5080"
+    - "6080"
+    volumes:
+    - type: bind
+      source: $GOPATH/bin
+      target: /gobin
+      read_only: true
+
+    command: /gobin/dgraph  ${COVERAGE_OUTPUT} zero --raft "idx=1;" --my=zero7_backup_clust_non_ha:5080 --replicas=1 --logtostderr -v=2 --bindall
+
+  alpha7_backup_clust_non_ha:
+    image: dgraph/dgraph:local
+    working_dir: /data/alpha7
+    links:
+      - "nfs:nfs"
+    privileged: true
+    labels:
+      cluster: test
+    ports:
+    - "8080"
+    - "9080"
+    volumes:
+    - type: bind
+      source: $GOPATH/bin
+      target: /gobin
+      read_only: true
+
+    command:
+      - /bin/sh
+      - -c
+      - |
+        apt-get update -qq && apt install nfs-client -y 
+        mount -v -o vers=4,loud nfs:/ /mnt 2>&1 & 
+        /gobin/dgraph  ${COVERAGE_OUTPUT}  alpha   --my=alpha7_backup_clust_non_ha:7080  --zero=zero7_backup_clust_non_ha:5080 --logtostderr -v=2  --security "whitelist=10.0.0.0/8,172.16.0.0/12,192.168.0.0/16;"
+
+          
+
+#non HA restore cluster
+
+  zero8_restore_clust_non_ha:
+    image: dgraph/dgraph:local
+    working_dir: /data/zero8
+    links:
+      - "nfs:nfs"
+    labels:
+      cluster: test
+    ports:
+    - "5080"
+    - "6080"
+    volumes:
+    - type: bind
+      source: $GOPATH/bin
+      target: /gobin
+      read_only: true
+
+    command: /gobin/dgraph  ${COVERAGE_OUTPUT} zero --raft "idx=1;" --my=zero8_restore_clust_non_ha:5080 --replicas=1 --logtostderr -v=2 --bindall
+
+  alpha8_restore_clust_non_ha:
+    image: dgraph/dgraph:local
+    working_dir: /data/alpha8
+    links:
+      - "nfs:nfs"
+    privileged: true
+    labels:
+      cluster: test
+    ports:
+    - "8080"
+    - "9080"
+    volumes:
+    - type: bind
+      source: $GOPATH/bin
+      target: /gobin
+      read_only: true
+    command:
+      - /bin/sh
+      - -c
+      - |
+        apt-get update -qq && apt install nfs-client -y 
+        mount -v -o vers=4,loud nfs:/ /mnt 2>&1 &
+        /gobin/dgraph  ${COVERAGE_OUTPUT}  alpha   --my=alpha8_restore_clust_non_ha:7080  --zero=zero8_restore_clust_non_ha:5080 --logtostderr -v=2  --security "whitelist=10.0.0.0/8,172.16.0.0/12,192.168.0.0/16;"
+
+  nfs:
+    image: itsthenetwork/nfs-server-alpine:${NFS_SERVER_IMAGE_ARCH:-12}
+    restart: unless-stopped
+    privileged: true
+    environment:
+      - SHARED_DIRECTORY=/data
+    volumes:
+      - /data/nfs-storage:/data
```

Comment:
check the error when we don't use this

Reply:
It is throwing this error "exportfs: Failed to stat /data: No such file or directory" if we remove volume.